README.md

```diff
@@ -1,95 +1,269 @@
+# jigawatt
+
 [![Build Status](https://travis-ci.org/influentialpublishers/jigawatt.svg?branch=master)](https://travis-ci.org/influentialpublishers/jigawatt)
 [![Coverage Status](https://coveralls.io/repos/github/influentialpublishers/jigawatt/badge.svg?branch=master)](https://coveralls.io/github/influentialpublishers/jigawatt?branch=master)
 [![codecov](https://codecov.io/gh/influentialpublishers/jigawatt/branch/master/graph/badge.svg)](https://codecov.io/gh/influentialpublishers/jigawatt)
 [![Code Climate](https://codeclimate.com/github/influentialpublishers/jigawatt/badges/gpa.svg)](https://codeclimate.com/github/influentialpublishers/jigawatt)
+[![standard-readme compliant](https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
-# jigawatt
 Influential's Functional, Promise-based Express Middleware
 
+### Table of Contents
+- [Installation](#installation)
+- [Using Jigawatt](#using-jigawatt)
+  - [Basic Usage](#basic-usage-example)
+  - [Middleware Structure](#middleware-structure)
+    - [awesomize](#awesomize)
+    - [io](#io)
+    - [transform](#transform)
+    - [Summary](#summary)
+- [Contribute](#contribute)
+- [License](#license)
+
 ## Installation
-`npm install jigawatt`
+`npm install jigawatt --save`
+
+## Using Jigawatt
+
+First, require the module.
 
-## Middleware Structure
+`const JW = require('jigawatt')`
 
-### Properties
-Jigawatt Middleware must consist of at least one of three properties:
-- `awesomize`: `(Validator -> AwesomizeSpec) -> Request -> Object a`
-  - validates/sanitizes the request to initialize the `data` object
-  - _for more info on `awesomize`, see https://www.npmjs.com/package/awesomize_
+For this walkthrough, we'll also be using Ramda.
 
-- `io`: `Request, Data -> Object a`
-  - returns an object that is merged into `req.data`
+`const R = require('ramda')`
 
-- `transform`: `Request, Data -> Object a`
-  - returns an object to become the new `req.data` value
+### Basic Usage Example
+
+For this example, we have a poll and poll responses stored in a database. Consider the following endpoint:
 
-### Example Middleware
 ```
-const _         = require('ramda')
-const Bluebird  = require('bluebird')
+app.get('/best-city-results/:id', (req, res) => {
```

Comment:
The code example should look something like so:
```javascript

const R = require('ramada') // use `R` for Ramda

app.get('/best-city-results/:id', JW(pollDetails, formatPoll)
```

Jigawatt middleware are *not* thenable (Promise) returning functions; they are simple javascript objects that have one or more of the following methods:  `awesomize`, `io` and/or `transform`.  If you would like to use a Jigawatt middleware as a Promise there is the `JW.promisify` method.